insta-clone/src/components/Authentication/Authentication.js

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import Login from '../Login/Login'
+
+const Authenticate = App => 
+    class extends React.Component {
+        constructor(props){
+            super(props);
+            this.state = {
+                loggedIn:false
+            };
+        }
+
+        componentDidMount() {
+            if(!localStorage.getItem('username')) {
+                this.setState({ loggedIn: false})
+            } else {
+                this.setState({loggedIn: true})
+            }
+        }
+
+        render() {
+           if(this.state.loggedIn) return <App />;
```

Comment:
You can use a ternary statement here: `this.state.loggedIn ? <App /> : <Login />`